chaos_genius/alerts/slack.py

```diff
@@ -61,8 +61,8 @@ def anomaly_alert_slack(
                 "text": {
                     "type": "mrkdwn",
                     "text": (
-                        f"- Total alerts generated (Overall KPI): *{overall_count}*\n"
-                        "- Total alerts generated (including subdimenions): "
+                        f"- Total alerts generated (Overall KPI): *{overall_count}*\n" +
```

Comment:
`+` on front looks more readable as aligned in same column, will leave this up to you